templates/master/01-master-container-runtime/_base/files/crio-seccomp-json.yaml

```diff
@@ -0,0 +1,241 @@
+mode: 0644
+path: "/etc/crio/crio-seccomp.json"
```

Comment:
What's the reason for adding it here as file in favor of having the profile in-memory: https://github.com/cri-o/cri-o/blob/af2035b294e34cfe64863922d985468d27d9de79/internal/config/seccomp/seccomp.go#L30 ?

Reply:
all encompassed in this PR to test :)

Reply:
Got it, thank you for the clarification.